nx-X11/programs/Xserver/randr/randr.c

```diff
@@ -309,7 +387,13 @@ RRFreeClient(void *data, XID id)
 
     pRREvent = (RREventPtr) data;
     pWin = pRREvent->window;
+#ifndef NXAGENT_SERVER
+    dixLookupResourceByType((void **) &pHead, pWin->drawable.id,
+                            RREventType, serverClient, DixDestroyAccess);
+#else                           /* !defined(NXAGENT_SERVER) */
     pHead = (RREventPtr *) LookupIDByType(pWin->drawable.id, RREventType);
+#endif                          /* !defined(NXAGENT_SERVER) */
+
```

Comment:
The patch you are referencing is part of the privates ABI/API change in X.org. The change is massive. For this RandR backport, I'd rather stick to the old approach handling privates in nx-libs.

However, I have also been playing with the thought of backporting the privates ABI/API, but this is a bunch of work and also includes introducing the Xace extension into nx-libs.

(Something we would gain while rebasing, anyway...)